File: p2-python-hola-mundo/p2-python-hola-mundo/perfume.py
# Importar la clase Perfumes
from perfumes import Perfumes

# Crear instancias de la clase Perfumes
perfume1 = Perfumes("born in Roma", "Valentino")
perfume2 = Perfumes("Scandal", "Jean Paul Gaultier")
perfume3 = Perfumes("Rave", "Lataffa")

# Llamar a los métodos para obtener el nombre y la marca
print(f"Perfume 1: Nombre - {perfume1.obtener_nombre()}, Marca - {perfume1.obtener_marca()}")
print(f"Perfume 2: Nombre - {perfume2.obtener_nombre()}, Marca - {perfume2.obtener_marca()}")
print(f"Perfume 3: Nombre - {perfume3.obtener_nombre()}, Marca - {perfume3.obtener_marca()}")

File: p3-java/practicajava/perfumes/Perfume.java
package perfumes;

public class Perfume {
    private String marca;
    private String nombre;
    
    public Perfume(String marca, String nombre) {
         this.marca = marca;
         this.nombre = nombre; 
        } 
        
    public void Mostrar() {
        System.out.println("====================");
        System.out.println("marca: " + this.marca);
        System.out.println("nombre: " + this.nombre);
        System.out.println("====================");
    } 
}